styles/component-styles/text-utilities.css

```diff
@@ -0,0 +1,40 @@
+.text-utilities-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
you can extract this into a utility class

Reply:
Yaa! good suggestion but then I have to make changes to all the components. I'll look into it after wrapping up with all assignments.